blockchain/chain_sync/src/peer_manager.rs

```diff
@@ -2,15 +2,19 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use async_std::sync::RwLock;
+use blocks::Tipset;
 use libp2p::core::PeerId;
 use log::debug;
+use std::collections::HashMap;
 use std::collections::HashSet;
 
 /// Thread safe peer manager
 #[derive(Default)]
 pub struct PeerManager {
     /// Hash set of full peers available
     full_peers: RwLock<HashSet<PeerId>>,
+    /// Represents peers and proposed tipsets from the network
+    pub peer_heads: RwLock<HashMap<PeerId, Tipset>>,
```

Comment:
why would you need both? peer_heads seems to be all you need (seems messy to handle update each seperately)

Reply:
Can you elaborate on what you mean by both? Are referencing having both peerId and tipsets?

Reply:
Nah I just mean you should only need the variable you created, the hash set isn't necessary (two copies of the peerids that have to be managed)

Reply:
I think I am going to remove this altogether from peer_manager, I only actually need the tipsets I believe